content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -87,22 +87,68 @@ sudo sysctl --system
 On Linux, {{< glossary_tooltip text="control groups" term_id="cgroup" >}}
 are used to constrain resources that are allocated to processes.
 
+Both {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} and the
+underlying container runtime need to interface with control groups to enforce
+[resource management for pods and
+containers](/docs/concepts/configuration/manage-resources-containers/) and set
+resources such as cpu/memory requests and limits. To interface with control
+groups, the kubelet and the container runtime need to use a *cgroup driver*. It's
+critical that both the kubelet and the container runtime cgroup driver match and
+are configured the same.
+
+There are two cgroup drivers available:
+
+* [`cgroupfs`](#cgroupfs-cgroup-driver)
+* [`systemd`](#systemd-cgroup-driver)
+
+### cgroupfs driver {#cgroupfs-cgroup-driver}
+
+The `cgroupfs` driver is the default cgroup driver in the kubelet. When  the `cgroupfs`
+driver is used, the kubelet and the container runtime directly interface with
+the cgroup filesystem to configure cgroups.
+
+The `cgroupfs` driver is **not** recommended when
+[systemd](https://www.freedesktop.org/wiki/Software/systemd/) is the
+init system because systemd expects a single cgroup manager on
+the system. Additionally, if you use [cgroup v2](/docs/concepts/architecture/cgroups)
+, use the `systemd` cgroup driver instead of
```

Comment:
```suggestion
the system. Additionally, if you use [cgroup v2](/docs/concepts/architecture/cgroups),
use the `systemd` cgroup driver instead of
```

Reply:
Is using `systemd` a must or a recommendation? Line 212 says it is a recommendation.

Reply:
It is a recommendation, but highly recommended :)